build.sh

```diff
@@ -18,7 +18,7 @@ else
 fi
 
 # Dependency revisions to use
-VCPKG_REV=9b44e4768bf25e1ae07e6eaba072c1a1a160f978
+VCPKG_REV=a62cad9bd3bc34316e103d738f1d72c62561b1f7
```

Comment:
Not sure when your change to vcpkg could land, but we likely don't want to commit this hash as script won't run on other machines due to missing revision?

Reply:
I've been waiting to get this merged into VCPKG before landing this for that reason. I hope vcpkg will land my PR (microsoft/vcpkg#10800) soon so I can merge the PRs here as well.